src/kernels/rq_iso.jl

```diff
@@ -27,35 +27,33 @@ get_param_names(rq::RQIso) = [:ll, :lσ, :lα]
 num_params(rq::RQIso) = 3
 
 metric(rq::RQIso) = SqEuclidean()
-cov(rq::RQIso, r::Float64) = rq.σ2*(1.0+r/(2.0*rq.α*rq.ℓ2)).^(-rq.α)
+cov(rq::RQIso, r::Float64) = rq.σ2*(1.0+r/(2.0*rq.α*rq.ℓ2))^(-rq.α)
 
-
-function grad_kern(rq::RQIso, x::Vector{Float64}, y::Vector{Float64})
-    r = distance(rq, x, y)
-    
-    g1 = rq.σ2*(r/rq.ℓ2)*(1.0+r/(2*rq.α*rq.ℓ2))^(-rq.α-1.0)       # dK_d(log ℓ)
-    g2 = 2.0*rq.σ2*(1+r/(2*rq.α*rq.ℓ2))^(-rq.α)                   # dK_d(log σ)
-    part = (1.0+r/(2*rq.α*rq.ℓ2))
-    g3 = rq.σ2*part^(-rq.α)*(r/(2*rq.ℓ2*part)-rq.α*log(part))*rq.α  # dK_d(log α)
-    return [g1,g2,g3]
-end
-
-
-function grad_stack!(stack::AbstractArray, rq::RQIso, X::Matrix{Float64}, data::IsotropicData)
-    nobsv = size(X,2)
+function addcov!(s::AbstractMatrix{Float64}, rq::RQIso, X::Matrix{Float64}, data::IsotropicData)
```

Comment:
This method is the same as that defined in stationary.jl. We'll delete this one unless there is a reason for keeping.